go.mod

```diff
@@ -18,19 +18,56 @@ require (
 )
 
 require (
+	github.com/beorn7/perks v1.0.1 // indirect
```

Comment:
auto generated by `go mod tidy -compat=1.17`

Reply:
I think these changes can be excluded from this PR, especially as our presubmits run with Go 1.18 now: https://github.com/jetstack/testing/blob/3bb9af8d51d14f32f6d3961a66e47e370d4d537e/config/jobs/cert-manager/csi-lib/cert-manager-csi-lib-presubmits.yaml#L16

(plus also exclude changes to go.sum as well)

Unless these came about due to the new tests you've added in this PR too? 👀

Reply:
Yes, these changes are due to new tests...required during `go test ./...`